go.mod

```diff
@@ -0,0 +1,77 @@
+module github.com/mainflux/mainflux
+
+go 1.13
+
+require (
+	github.com/Azure/go-ansiterm v0.0.0-20170929234023-d6e3b3328b78 // indirect
+	github.com/BurntSushi/toml v0.3.1
+	github.com/Microsoft/go-winio v0.4.7 // indirect
+	github.com/Nvveen/Gotty v0.0.0-20120604004816-cd527374f1e5 // indirect
+	github.com/VividCortex/gohistogram v1.0.0 // indirect
+	github.com/bitly/go-hostpool v0.0.0-20171023180738-a3a6125de932 // indirect
+	github.com/bmizerany/assert v0.0.0-20160611221934-b7ed37b82869 // indirect
+	github.com/cenkalti/backoff v2.0.0+incompatible // indirect
+	github.com/cisco/senml v0.0.0-20181031221301-910a55054e16
+	github.com/codahale/hdrhistogram v0.0.0-20161010025455-3a0bb77429bd // indirect
+	github.com/containerd/continuity v0.0.0-20180416230128-c6cef3483023 // indirect
+	github.com/dgrijalva/jwt-go v3.2.0+incompatible
+	github.com/docker/docker v1.13.1
+	github.com/docker/go-connections v0.3.0 // indirect
+	github.com/docker/go-units v0.3.3 // indirect
+	github.com/dustin/go-coap v0.0.0-20170214053734-ddcc80675fa4
+	github.com/eclipse/paho.mqtt.golang v1.1.1
+	github.com/fatih/color v1.7.0
+	github.com/go-kit/kit v0.8.0
+	github.com/go-redis/redis v6.15.0+incompatible
+	github.com/go-zoo/bone v0.0.0-20160911183509-fd0aebc74e90
+	github.com/gobuffalo/packr v1.30.1 // indirect
+	github.com/gocql/gocql v0.0.0-20181106112037-68ae1e384be4
+	github.com/gofrs/uuid v3.2.0+incompatible
+	github.com/gogo/protobuf v1.2.1
+	github.com/golang/protobuf v1.3.1
+	github.com/golang/snappy v0.0.0-20180518054509-2e65f85255db // indirect
+	github.com/gorilla/websocket v1.4.0
+	github.com/gotestyourself/gotestyourself v2.2.0+incompatible // indirect
+	github.com/hokaccha/go-prettyjson v0.0.0-20180920040306-f579f869bbfe
+	github.com/influxdata/influxdb v1.6.4
+	github.com/jmoiron/sqlx v0.0.0-20190319043955-cdf62fdf55f6
+	github.com/lib/pq v1.0.0
+	github.com/magiconair/properties v1.8.1 // indirect
+	github.com/mattn/go-colorable v0.0.9 // indirect
+	github.com/mattn/go-isatty v0.0.3 // indirect
+	github.com/nats-io/gnatsd v1.4.1 // indirect
+	github.com/nats-io/go-nats v1.6.0
+	github.com/nats-io/nuid v1.0.0 // indirect
+	github.com/onsi/ginkgo v1.10.1 // indirect
+	github.com/onsi/gomega v1.7.0 // indirect
+	github.com/opencontainers/go-digest v1.0.0-rc1 // indirect
+	github.com/opencontainers/image-spec v1.0.1 // indirect
+	github.com/opencontainers/runc v0.1.1 // indirect
+	github.com/opentracing/opentracing-go v1.1.0
+	github.com/ory/dockertest v3.3.0+incompatible // indirect
+	github.com/pelletier/go-toml v1.4.0 // indirect
+	github.com/prometheus/client_golang v0.9.3
+	github.com/rubenv/sql-migrate v0.0.0-20181106121204-ba2c6a7295c5
+	github.com/sony/gobreaker v0.0.0-20180905101324-b2a34562d02c
```

Comment:
I recommend that you change this one to version `0.4.1`

Reply:
@anovakovic01 Done